Source/Core/DolphinQt2/MainWindow.cpp

```diff
@@ -191,6 +182,14 @@ void MainWindow::ScreenShot()
 	Core::SaveScreenShot();
 }
 
+void MainWindow::PathsConfig()
+{
+	PathDialog* paths = new PathDialog(this);
```

Comment:
I think the previous PathDialog needs to be reused here otherwise we end up with multiple hidden PathDialogs. Alternatively you can set Qt::WA_DeleteOnClose to make Qt delete the PathDialog when it is closed, but I cant see that used either.